package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@elastic/snyk-github-issue-creator",
-    "version": "2.1.1",
+    "version": "2.1.2",
```

Comment:
@watson Is there anything else I need to do to support rolling out a new version?

Reply:
The version number should be updated independently of the changes in a PR. So just revert any changes to this file in this PR. To prepare and publish a new version, do the following:

```sh
# ensure you have the latest main checked out locally
git pull upstream main
# bump the patch version in package.json and create version commit + tag
npm version patch
# ensure the patch commit is pushed upstream
git push upstream main
# ensure the version tag is pushed upstream
git push upstream --tags
# publish the new version to npm
npm publish
```